src/api/ingestion.ts

```diff
@@ -7,7 +7,7 @@ function parsePdfIngestionResponse(response: IngestResponse): ReferenceItem[] {
   return response.references.map((reference) => ({
     id: reference.filename_md5,
     title: reference.title ?? reference.source_filename.replace('.pdf', ''),
-    authors: (reference.authors ?? []).map((author) => ({ fullName: author.full_name, surname: author.surname })),
+    authors: (reference.authors ?? []).map((author) => ({ fullName: author.full_name })),
```

Comment:
@cguedes @sehyod I modified this to remove `surname` since it is not guaranteed we will have it even though we might have `reference.authors`. Feel free to push a commit if there's a better way to do this!